.github/workflows/build.yml

```diff
@@ -71,6 +89,5 @@ jobs:
 
       - run: yarn install --frozen-lockfile
 
-      # `&` symbol is needed as it prevents Storybook from running endlessly
-      - run: yarn dev &
-      - run: yarn workspace @itwin/itwinui-react creevey
+      - run: yarn build --filter=storybook
```

Comment:
What is the difference between running commands like this versus `yarn workspace...`?